### PR TITLE
reset text input after editing message

### LINKED ISSUE #12277: After editing message sticker and audio buttons are not shown in chat input until reopening chat view
# Bug Report

## Problem 
When sending edited message, buttons "Audio" and "Stikers" are not shown in chat input.
To make them display you need to type smth and then clean chat input or reopen chat view.

#### Expected behavior
can see chat input with  "Audio" and "Stikers" after message sending

#### Actual behavior
https://user-images.githubusercontent.com/4557972/124473907-29292400-dda0-11eb-8c11-6dbaa6aa476a.mp4

### Reproduction

1) Open Status
2) Open 1-1 chat
3) Send message
4) Edit message and pay attention to chat input after editing

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by bitgamma**
Fixes #12277

### CODE REVIEW

File: src/status_im/chat/models/input.cljs [hunk 1]
```diff
@@ -235,7 +235,8 @@
                       :params     [{:id message-id :text text}]
                       :js-response true
                       :on-error #(log/error "failed to edit message " %)
-                      :on-success #(re-frame/dispatch [:sanitize-messages-and-process-response %])}]}
+                      :on-success #(re-frame/dispatch [:sanitize-messages-and-process-response %])}]
+    :set-input-text [(:current-chat-id db) ""]}
```
Comment (flexsurfer) [inline src/status_im/chat/models/input.cljs]: why not `cancel-message-edit` ? it seems we need to clear more ?

Comment (cammellos) [reply src/status_im/chat/models/input.cljs]: if we can re-use cancel-message-edit and it works, I would probably use that, as it's very fiddly to clean the input text

Comment (bitgamma) [reply src/status_im/chat/models/input.cljs]: thanks, will try

Comment (bitgamma) [reply src/status_im/chat/models/input.cljs]: seems to work fine

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**